nss-day-cohort-46/holiday-road-expired-destruction
# Major Css overhaul so the site doesn't look like hot garbage. I didn'…

…t break anything I promise.

# Description

I did a major overhaul of the css to make the site look a little more phisticated.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

Please check this in your browser.

```
git fetch all
git checkout EB-Css-Fixes
```



# Checklist:
